docs/guides/deployment/fly_io.rst

```diff
@@ -82,6 +87,59 @@ default Fly.io VM side provides:
           CPU Cores: 1
              Memory: 1 GB
 
+Proxying connections
```

Comment:
I would make this a subsection of "Connecting to the instance" and name it "From external application"